requirements.txt

```
@@ -0,0 +1,4 @@
+torch
```

requirements.txt нужно перенести в проект, у каждого проекта он свой, по сути каждая из частей - это отдельные проекты. В идеале, тут заводится групповой репозиторий, но т.к. гитхаб не дает такой возможности из коробки,
то приходится помещать все проекты в 1 репозиторий

project/
```suggestion
torch
```
    src/
        <code.py>
    tests/
        test_
    params/
       config's...
    tasks/
       sh задачи
    requirements.txt
   .env - переменные окружения, если нужны
    README
    остальные файлы

Считаем, что любой запуск идет из корня